server/src/server.ts

```diff
@@ -774,6 +801,7 @@ function onMessage(msg: p.Message) {
     } else if (msg.method === DidOpenTextDocumentNotification.method) {
       let params = msg.params as p.DidOpenTextDocumentParams;
       openedFile(params.textDocument.uri, params.textDocument.text);
+      updateDiagnosticSyntax(params.textDocument.uri, params.textDocument.text);
```

Comment:
@zth  is this the correct place?

Reply:
Want to be up to date for each edit of the file.

Reply:
Yes, I think they're in the right places. They'll run on each file open and on changes to the documents. I think that makes sense.